.gitmodules

```diff
@@ -4,9 +4,9 @@
 [submodule "dump978"]
 	path = dump978
 	url = https://github.com/ankurvdev/dump978
-[submodule "libusb"]
-	path = libusb
-	url = https://github.com/ankurvdev/libusb
-[submodule "rtl-sdr"]
-	path = rtl-sdr
-	url = https://github.com/osmocom/rtl-sdr
+[submodule "libusb-1.0"]
+	path = libusb-1.0
+	url = https://github.com/libusb/libusb.git
+[submodule "librtlsdr"]
```

Comment:
This seems to be older and less actively maintained than the osmocom one which is the official version.

Any reason for the change ?